src/hm-admin/hm-admin.component.ts

```diff
@@ -7,5 +8,13 @@ import { Component } from '@angular/core';
   styleUrls: ['hm-admin.component.css']
 })
 export class HmAdminComponent {
+  @Input('MyCustomConfig') MyCustomConfig: any;
```

Comment:
```
@Input('my-custom-config')
MyCustomConfig: any;
```
is better.

the parameter allow you to rename the input.

BTW, `config` is a better name than `my-custom-config` ;)